sdk/daml-assistant/src/DA/Daml/Assistant/Version.hs

```diff
@@ -594,6 +594,16 @@ osName = case System.Info.os of
     "mingw32" -> "windows"
     p -> error ("daml: Unknown operating system " ++ p)
 
+-- | Architecture-specific part of the asset name
+archName :: Text
+archName = case System.Info.arch of
+    "x86_64"   -> "x86_64"
+    "aarch64" -> "aarch64"
+    p -> error ("daml: Unknown architecture " ++ p)
```

Comment:
We're sure this won't cause issues anywhere? i.e. a 32 bit machine?

Reply:
I was thinking the same thing when writing this - we don't have any 32-bit executables as far as I'm aware, so a 32-bit machine would not be able to run the 64-bit executables we've produced anyways. Am I completely off the mark here?